datahub-web-react/src/app/entity/user/UserEditProfileModal.tsx

```diff
@@ -0,0 +1,221 @@
+                        value={userImageURL}
+                        onChange={(event) => setImageURL(event.target.value)}
+                    />
+                </Form.Item>
+                <Form.Item
+                    name="team"
+                    label={<Typography.Text strong>Team</Typography.Text>}
+                    rules={[{ whitespace: true }, { min: 2, max: 50 }]}
+                >
+                    <Input
+                        placeholder="add team name"
+                        value={userTeam}
+                        onChange={(event) => setUserTeam(event.target.value)}
+                    />
+                </Form.Item>
+                <Form.Item
+                    name="email"
+                    label={<Typography.Text strong>Email</Typography.Text>}
+                    rules={[
+                        {
+                            required: true,
+                            message: 'Please enter your email',
+                        },
+                        {
+                            type: 'email',
+                            message: 'Please enter valid email',
+                        },
+                        { whitespace: true },
+                        { min: 2, max: 50 },
+                    ]}
+                    hasFeedback
+                >
+                    <Input
+                        placeholder="add email"
+                        value={userEmail}
+                        onChange={(event) => setUserEmail(event.target.value)}
+                    />
+                </Form.Item>
+                <Form.Item
+                    name="slack"
+                    label={<Typography.Text strong>Slack</Typography.Text>}
+                    rules={[{ whitespace: true }, { min: 2, max: 50 }]}
+                    hasFeedback
+                >
+                    <Input
+                        placeholder="add slack id"
+                        value={userSlack}
+                        onChange={(event) => setUserSlack(event.target.value)}
+                    />
+                </Form.Item>
+                <Form.Item
+                    name="phone"
+                    label={<Typography.Text strong>Phone</Typography.Text>}
+                    rules={[
+                        {
+                            pattern: new RegExp('^(?=.*[0-9])[- +()0-9]+$'),
+                            message: 'not valid phone number',
+                        },
+                        {
+                            min: 5,
+                            max: 15,
+                        },
+                    ]}
+                    hasFeedback
+                >
+                    <Input
+                        placeholder="add phone number"
+                        value={userPhoneNumber}
+                        onChange={(event) => setUserPhoneNumber(event.target.value)}
+                    />
+                </Form.Item>
+            </Form>
+        </Modal>
```

Comment:
Summary: Each placeholder should generally be an example of what goes inside the placeholder.

Reply:
sure